js/controller.js

```diff
@@ -500,12 +500,13 @@ module.exports = function(OO, _, $, W) {
     toggleAirPlayIcon: function(event) {
       if (!this.state.airPlayStatusIcon) {
         const airPlayState = window.sessionStorage.getItem('airPlayState');
-        if (airPlayState) {
-          this.state.airPlayStatusIcon = airPlayState;
-          this.state.mainVideoElement.webkitShowPlaybackTargetPicker();
-        } else {
-          this.state.airPlayStatusIcon = 'airPlay-disconnected';
+        if (airPlayState === 'airPlay-connected') {
```

Comment:
I guess we should keep `airPlay-connected` as a constant

Reply:
Fixed